api/api.py

```diff
@@ -520,7 +520,7 @@ def encode_yaml_stream(objs: List[StructuredDataType]) -> Blob:
   """
   pass
 
-def default_registry(registry: str) -> None:
+def default_registry(registry: str, host_from_cluster: str = None) -> None:
```

Comment:
while you're here, i think the first param is called `host`?
https://github.com/windmilleng/tilt/blob/59de82aedb3bad31a65524b5e8d3279eb8650231/internal/tiltfile/docker.go#L519
using a named parameter `registry` won't work

Reply:
whoops good catch